src/main/java/com/dft/prestashop/PrestashopAccessCredentials.java

```diff
@@ -0,0 +1,15 @@
+package com.dft.prestashop;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Remove @NoArgsConstructor
@AllArgsConstructor

Reply:
if i am removing this it does not giving me argument constructor
![error1](https://github.com/dftrakesh/prestashop/assets/159010762/a0852e13-2995-46d3-8b2e-44211a87cb57)

Reply:
done